android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -448,15 +450,13 @@ public void onConfigurationChanged(Configuration newConfig) {
     if (this.htmlBannerString != null && !this.htmlBannerString.isEmpty()) {
       setHTMLBanner(this.htmlBannerString);
     }
-    loadJavascript(KMString.format("setBannerHeight(%d)", bannerHeight));
-    loadJavascript(KMString.format("setOskWidth(%d)", newConfig.screenWidthDp));
-    loadJavascript(KMString.format("setOskHeight(%d)", oskHeight));
 
-    this.dismissHelpBubble();
+    DisplayMetrics dms = context.getResources().getDisplayMetrics();
+    int kbWidth = (int) (dms.widthPixels / dms.density);
```

Comment:
This specific line - `kbWidth` - has two other instances in the codebase already.  This simply makes a third.